pytest.ini

```diff
@@ -1,5 +1,6 @@
 [pytest]
 addopts = -W ignore
+          --durations=100
```

Comment:
This is duplicated in PR #54, isn't it?

Reply:
Yes, sorry, I think I left that change in when starting on #54 by mistake. Given the type of change I don't think there should be any issue (we should be able to e.g. merge these just one after the other), but I can also revert it there (or close this PR) if this would make things neater.